src/components/DiaryHome.js

```diff
@@ -77,10 +80,18 @@ const DiaryHome = ( { onAdd }) => {
                     SUBMIT
                 </Button>
             </form>
-            {/* {notes.length > 0 ? (<DiaryCard tittle='Hello world' subtitle='subtitle' color='red' description='simple text'/>) : ('No Notes')} */}
         </>
      );
 }
+
+const mapDispatchToProps = dispatch => {
+    return {
+        add_new_note: ({id, tittle , description})=>dispatch({
+            type:"ADD_NOTE",
+            payload: {id,tittle , description}
+        })
+    }
+}
  
-export default DiaryHome;
+export default connect(null,mapDispatchToProps)(DiaryHome);
```

Comment:
this method is ok but this is an old method to dispatch actions. You can use useDispatch and useSelector hooks.